sourcegraph/sourcegraph
# [SG-43143] Expanding search matches should focus the first previously-hidden match, not the collapse button

### Description
For content match search results, expanding the "Show $x more matches" keeps focus on that button (which now reads "Show less". This is a problem because keyboard users have to tab back and navigate the new matches in reverse order.

### Refs
[Sourcegraph Issue](https://github.com/sourcegraph/sourcegraph/issues/43143)
[Gitstart Ticket](https://app.gitstart.com/clients/sourcegraph/tickets/SG-43143)

### Expected Behavior
The first previously-hidden match should be focus on expansion.

### Test Plan / Loom Video
1. On search list page, identify a search term with hidden match
2. Click to expand the hidden matches
3. Verify that the focus is on the first previously-hidden match

### DEMO
https://www.loom.com/share/9c7af8faa1ec4f619885baadc2f029c0

## App preview:

- [Web](https://sg-web-contractors-sg-43143.onrender.com/search)

Check out the [client app preview documentation](https://docs.sourcegraph.com/dev/how-to/client_pr_previews) to learn more.
